src/Resources/views/standard_layout.html.twig

```diff
@@ -96,7 +96,7 @@ file that was distributed with this source code.
                             {% endif %}
 
                             {%- set translation_domain = menu.extra('translation_domain', 'messages') -%}
-                            {%- set label = menu.label -%}
+                            {%- set label = menu.getLabel() -%}
```

Comment:
It's common to use the accessor syntax instead of the explicit method call. A comment explaining the reason will help to avoid this change to be reverted in the future.

Reply:
:+1: